process/rating/peerHonesty/peerScore_test.go

```diff
@@ -0,0 +1,418 @@
+package peerHonesty
+
+import (
+	"errors"
+	"sync/atomic"
+	"testing"
+	"time"
+
+	"github.com/ElrondNetwork/elrond-go/config"
+	"github.com/ElrondNetwork/elrond-go/core/check"
+	"github.com/ElrondNetwork/elrond-go/process"
+	"github.com/ElrondNetwork/elrond-go/process/mock"
+	"github.com/ElrondNetwork/elrond-go/testscommon"
+	"github.com/stretchr/testify/assert"
+)
+
+// createMockPeerHonestyConfig creates a peer honesty config with reasonable values
+func createMockPeerHonestyConfig() config.PeerHonestyConfig {
+	return config.PeerHonestyConfig{
+		DecayCoefficient:             0.9995,
```

Comment:
I know I am exaggerating here, but can you use the same value as the one from `config.toml` perhaps?

Reply:
done